step3/app.js

```diff
@@ -0,0 +1,77 @@
+const express = require('express');
+const fs = require('fs');
+const app = express();
+
+PORT = 3000;
+
+const getItems = () => {
+    const getItems = JSON.parse(fs.readFileSync('./data.json','utf-8'));
+    return getItems;
+}
+
+const save = (items) => {
+    fs.writeFileSync('./data.json',JSON.stringify(items),'utf-8');
+}
+
+function updateIdx(items) {
+    for(let i =0;i<items.length;++i) {
+        items[i].idx = i + 1; 
+    }
+}
```

Comment:
```suggestion
function updateIdx(items) {
    items.forEach((v, k) => {
      items[k].idx = k + 1;
    })
}
```
이렇게 array 관련 method를 이용해주면 더 직관적으로 표현해줄 수 있어요!